backend/internal/handlers/webhooks/webhooks.go

```diff
@@ -76,6 +76,14 @@ func (s *WebHookService) PushEvent(c *fiber.Ctx) error {
 		}
 	}
 
+	// If students pushed commits, update the work state accordingly
+	if pushEvent.Pusher != nil && *pushEvent.Pusher.Name != "khoury-classroom[bot]" && pushEvent.Commits != nil && len(pushEvent.Commits) > 0 {
```

Comment:
hard coding the name of the github app here probably isn't the best, especially since we most likely will change it. However, it might be overkill to go and change every instance of it in our app in this PR

Reply:
Agree that it feels clunky. Perhaps, rather than hard coding, I check if the string has `[bot` in it. Either way, this is non-breaking, just attempts to refine the commit counter.